pyrometheus/__init__.py

```diff
@@ -555,7 +555,10 @@ def gen_python_code(sol: ct.Solution):
                                            [isinstance(r, ct.ThreeBodyReaction)
                                             for r in sol.reactions()])),
     )
-    print(code)
+    if file is not None:
+        print(code, file=file)
+    else:
+        print(code)
     exec_dict = {}
     exec(compile(code, "<generated code>", "exec"), exec_dict)
```

Comment:
```suggestion
    exec(compile(code, "<generated code>", "exec"), exec_dict)
    exec_dict["_MODULE_SOURCE_CODE"] = code
```
Using this, you can see the generated code in https://documen.tician.de/pudb/.

Reply:
This was more about getting the actual py file than it was about putting eyes on the code.  It isn't essential.